app/Http/Controllers/Administration/SettingsController.php

```diff
@@ -241,6 +241,24 @@ public function setMapDisplayPublic(Request $request)
 		return Configs::set('map_display_public', '0') ? 'true' : 'false';
 	}
 
+	/**
+	 * Enable display of photo direction on map.
+	 *
+	 * @param Request $request
+	 *
+	 * @return string
+	 */
+	public function setMapDisplayDirection(Request $request)
```

Comment:
Are you planning to add a GUI to the settings to control this? I don't see one in your front end PR so far, so I don't think you need this at all?

Reply:
Ah yeah, that was leftover from when I was experimenting to see where things were set in the engine. I don't think this needs to be exposed in the GUI but obviously it could be if people think it would be useful. Should I remove the backend code then?

Reply:
Yeah, adding a GUI is a PITA and personally I wouldn't bother. Since this is effectively dead code, I would recommend removing it.

Reply:
Done!